romancal/jump/jump_step.py

```diff
@@ -5,8 +5,9 @@
 import roman_datamodels as rdm
 import numpy as np
 import time
+
 from ..stpipe import RomanStep
-from .. roman_datamodels import dqflags
+from .. datamodels import dqflags
```

Comment:
We need to agree on a place for dqflags, perhaps `romancal/lib`.

Reply:
`jwst/datamodels` are still in jwst, though there's a request to make them a standalone package. If/when this is done I suspect the module will be moved.
`romancal/datamodels` needs to be removed since the datamodels are a standalone package from the start. That's why `dqflags` needs to go in some other "common" directory. 
#262 moved `dqflags` to `lib`, import the module from there.